gax-grpc/src/test/java/com/google/api/gax/grpc/SettingsTest.java

```diff
@@ -56,13 +56,13 @@
 import com.google.common.collect.Lists;
 import com.google.common.truth.Truth;
 import java.io.IOException;
-import org.apache.commons.lang3.builder.EqualsBuilder;
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.rules.ExpectedException;
 import org.junit.runner.RunWith;
 import org.junit.runners.JUnit4;
 import org.mockito.Mockito;
+import org.mockito.internal.matchers.apachecommons.ReflectionEquals;
```

Comment:
I dont think we should be using classes from mockito's internal pacakge

Reply:
To remove the dependency over common-lang. Should implementing a utility similar to `EqualsBuilder.reflectionEquals` would be ok?

I think there isn't any equivalent utility present in guava as of now.